Docs/developer_guide/python_faq.md

```diff
@@ -348,8 +348,21 @@ In our example:
 
 ## Can I use any Python package in a Slicer module
 
-Any Python packages can be installed in Slicer's built-in Python environment. A convenience function `slicer.util.pip_install` can be used as shown in the [example in the Script Repository](script_repository.md#install-a-python-package).
```

Comment:
@jamesobutler  I just realized I forgot to list you as a co-author on this commit :disappointed:

Reply:
No worries!